lib/active_record/virtual_attributes/virtual_arel.rb

```diff
@@ -11,7 +11,13 @@ module VirtualAttributes
     # Model.attribute_supported_by_sql?(:field) # => true
 
     # in essence, this is our Arel::Nodes::VirtualAttribute
-    class Arel::Nodes::Grouping
+    class ::Arel::Nodes::VirtualAttribute < Arel::Nodes::Grouping
```

Comment:
It would have been nice if our attribute was in our namespace, but Arel requires that all nodes be in their namespace

Reply:
hmm, that's too bad.  Do you think they don't want this exposed?  I wonder if there's another way.  Can we use composition with a module providing the behavior instead?  I'd be afraid of adding to their namespace.  I guess we'd still be including a module in their namespace.  Maybe, it's moot then.

Reply:
YAY - thank you. This works in our own namespace.

fixed